apps/project/tests/test_mutations.py

```diff
@@ -265,17 +265,22 @@ def _query_check(**kwargs):
         self.assertMatchSnapshot(response, 'private-af-private-project-success')
 
         # Valid [public AF] + private project
+        Project.objects.get(id=response['result']['id']).delete()
```

Comment:
Can we change the title instead of deleting the project?

Reply:
> Can we change the title instead of deleting the project?

if we change the title the test fails for shapshot, ie self.assertMatchSnapshot(response, 'public-af-private-project-success') @thenav56